src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -797,4 +797,27 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         return winrt::single_threaded_vector<Model::Command>(std::move(result));
     }
+
+    IVector<Model::Command> Command::ToSendInputCommands(IVector<hstring> commands)

```

Comment:
This is basically `HistoryToCommands(commands, L"", false)`, but then with a different icon, yea? Seems like it'd be easier to stick an optional icon as a parameter on `HistoryToCommands`